modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/Godot.SourceGenerators.Tests.csproj

```diff
@@ -2,9 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-
-    <LangVersion>11</LangVersion>
```

Comment:
We do use raw string literals, but it shouldn't be hard to change if we want to. The syntax is just a tad cleaner.

Reply:
Oh, I love raw string literals. Since we use C# 11 features then this change is wrong. Unfortunately, because we don't run the tests in CI, the checks pass even though it would break the project.

Reply:
Could we enable the dotnet tests in CI?

Reply:
It shouldn't be too complicated. I'm not familiar with what our standards from actions / workflows are, but it's basically calling `dotnet test` from the test folder.